meta/system.py

```diff
@@ -287,6 +287,26 @@ def gt_features(self, corpus, prefix="", **kwargs):
         self.feature_flows[corpus]["gt"] = features.basic_cache_flow(feature_path)
         self.feature_flows[corpus]["uncached_gt"] = f.feature_flow
 
+    def generic_features(
+        self, corpus, name, feature_flow, port_name_mapping, prefix="", **kwargs
```

Comment:
you could add a default port_name mapping, often the output of a flow network is "features"